file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [18-May-2025 06:55:12 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>
            create table wp_dtables_tables (
                id bigint unsigned not null auto_increment primary key,
                title varchar(255) not null,
                type varchar(255) not null,
                datasource longtext null,
                options longtext null,
                meta longtext null,
                created_at timestamp null,
                updated_at timestamp null
            ) </p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown create type: table.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/dtables/class-install.php(71): WP_SQLite_DB->query('\n            cr...')
#2 /wordpress/wp-content/plugins/dtables/class-install.php(32): dtables\Install->create_tables()
#3 /wordpress/wp-content/plugins/dtables/class-app.php(191): dtables\Install->install()
#4 /wordpress/wp-includes/class-wp-hook.php(324): dtables\App->activate(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_dtable...', false)
#8 /internal/eval.php(20): activate_plugin('dtables/dtables...')
#9 {main}</pre>
 for query 
            create table wp_dtables_tables (
                id bigint unsigned not null auto_increment primary key,
                title varchar(255) not null,
                type varchar(255) not null,
                datasource longtext null,
                options longtext null,
                meta longtext null,
                created_at timestamp null,
                updated_at timestamp null
            )  made by activate_plugin, do_action('activate_dtables/dtables.php'), WP_Hook->do_action, WP_Hook->apply_filters, dtables\App->activate, dtables\Install->install, dtables\Install->create_tables, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[18-May-2025 06:55:21 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>
            create table wp_dtables_tables (
                id bigint unsigned not null auto_increment primary key,
                title varchar(255) not null,
                type varchar(255) not null,
                datasource longtext null,
                options longtext null,
                meta longtext null,
                created_at timestamp null,
                updated_at timestamp null
            ) </p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown create type: table.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/dtables/class-install.php(71): WP_SQLite_DB->query('\n            cr...')
#2 /wordpress/wp-content/plugins/dtables/class-install.php(32): dtables\Install->create_tables()
#3 /wordpress/wp-content/plugins/dtables/class-app.php(191): dtables\Install->install()
#4 /wordpress/wp-includes/class-wp-hook.php(324): dtables\App->activate(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_dtable...', false)
#8 /internal/eval.php(20): activate_plugin('dtables/dtables...')
#9 {main}</pre>
 for query 
            create table wp_dtables_tables (
                id bigint unsigned not null auto_increment primary key,
                title varchar(255) not null,
                type varchar(255) not null,
                datasource longtext null,
                options longtext null,
                meta longtext null,
                created_at timestamp null,
                updated_at timestamp null
            )  made by activate_plugin, do_action('activate_dtables/dtables.php'), WP_Hook->do_action, WP_Hook->apply_filters, dtables\App->activate, dtables\Install->install, dtables\Install->create_tables, WP_SQLite_DB->query, WP_SQLite_DB->print_error

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #5 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"duplicate-plugin"},"options":{"activate":true}}) : Could not download "https://downloads.wordpress.org/plugin/duplicate-plugin.latest-stable.zip".
  				Check if the URL is correct and the server is reachable.
  				If it is reachable, the server might be blocking the request.
  				Check the browser console and network tabs for more information.
  
  				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
  
  				This means the server that hosts your file does not allow requests from other sites
  				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
  				cross-origin file downloads. Learn more about CORS at
  				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
  
  				If your file is on GitHub, load it from "raw.githubusercontent.com".
  				Here's how to do that:
  
  				1. Start with the original GitHub URL of the file. For example:
  				https://github.com/username/repository/blob/branch/filename.
  				2. Replace "github.com" with "raw.githubusercontent.com".
  				3. Remove the "/blob/" part of the URL.
  
  				The resulting URL should look like this:
  				https://raw.githubusercontent.com/username/repository/branch/filename
  
  				Error:
  				Error: Could not download "https://downloads.wordpress.org/plugin/duplicate-plugin.latest-stable.zip"
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: Error: Could not download "https://downloads.wordpress.org/plugin/duplicate-plugin.latest-stable.zip".
    				Check if the URL is correct and the server is reachable.
    				If it is reachable, the server might be blocking the request.
    				Check the browser console and network tabs for more information.
    
    				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
    
    				This means the server that hosts your file does not allow requests from other sites
    				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
    				cross-origin file downloads. Learn more about CORS at
    				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
    
    				If your file is on GitHub, load it from "raw.githubusercontent.com".
    				Here's how to do that:
    
    				1. Start with the original GitHub URL of the file. For example:
    				https://github.com/username/repository/blob/branch/filename.
    				2. Replace "github.com" with "raw.githubusercontent.com".
    				3. Remove the "/blob/" part of the URL.
    
    				The resulting URL should look like this:
    				https://raw.githubusercontent.com/username/repository/branch/filename
    
    				Error:
    				Error: Could not download "https://downloads.wordpress.org/plugin/duplicate-plugin.latest-stable.zip"
        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)
  }
}

Node.js v23.11.0
